crates/humanode-runtime/src/currency_swap.rs

```diff
@@ -0,0 +1,48 @@
+use bridge_pot_currency_swap::{CurrencySwap, ExistenceRequired};
+use sp_runtime::traits::Identity;
+
+use crate::{
+    parameter_types, AccountId, Balances, EvmAccountId, EvmBalances, EvmBalancesPot,
+    NativeToEvmSwapBridgePot,
+};
+
+parameter_types! {
+    pub NativeToEvmSwapBridgePotAccountId: AccountId = NativeToEvmSwapBridgePot::account_id();
+    pub EvmBalancesPotAccountId: EvmAccountId = EvmBalancesPot::account_id();
+}
+
+pub type NativeToEvmOneToOne = CurrencySwap<NativeToEvmOneToOneConfig, ExistenceRequired>;
+
+pub struct NativeToEvmOneToOneConfig;
+
+impl bridge_pot_currency_swap::Config for NativeToEvmOneToOneConfig {
+    type AccountIdFrom = AccountId;
+    type AccountIdTo = EvmAccountId;
+    type CurrencyFrom = Balances;
+    type CurrencyTo = EvmBalances;
+    type BalanceConverter = Identity;
+    type PotFrom = NativeToEvmSwapBridgePotAccountId;
+    type PotTo = EvmBalancesPotAccountId;
+}
+
+pub type EvmToNativeOneToOne = CurrencySwap<EvmToNativeOneToOneConfig, ExistenceRequired>;
```

Comment:
```suggestion
pub type EvmToNativeOneToOne = bridge_pot_currency_swap::CurrencySwap<EvmToNativeOneToOneConfig, ExistenceRequired>;
```

Let's be more explicit here

Reply:
https://github.com/humanode-network/humanode/pull/673/commits/ff44e47d349d471c9cf8b2e53b4a9818fcc4c2c1